src/app/pages/HomePage/HomePage.tsx

```diff
@@ -3,11 +3,12 @@
 import React, { useContext } from 'react';
 import { jsx } from '@emotion/react';
 import {
-  CurationData,
   VisualProminence,
   VisualStyle,
-} from '#app/models/types/curationData';
-import Curation from '#app/components/Curation';
+  CurationData,
+} from '../../models/types/curationData';
+import Curation from '../../components/Curation';
+import VisuallyHiddenText from '../../legacy/psammead/psammead-visually-hidden-text/src';
```

Comment:
To counter the changes about the components standards in the other PR 😅 , this component actually looks pretty small and could be a quick win to bring it in to main `components` folder.

Its likely we will be using it for the likes of Live pages as well. Doesn't need to be part of this, but may be a good one to convert over?

Reply:
We have allowed a little bit of extra time this sprint to do this. Let me do it in a separate PR, and use it here. Thanks for the suggestion.

Reply:
This has now been completed as part of #10745